libs/librepcb/libraryeditor/newelementwizard/newelementwizardpage_entermetadata.ui

```diff
@@ -121,6 +121,22 @@
        </property>
       </widget>
      </item>
+     <item>
+      <widget class="QLabel" name="lblMoreCategories">
+       <property name="sizePolicy">
+        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
+         <horstretch>0</horstretch>
+         <verstretch>0</verstretch>
+        </sizepolicy>
+       </property>
+       <property name="text">
+        <string>... and X more.</string>
```

Comment:
Mark this string as not translatable to avoid unnecessary effort for translators:

```suggestion
        <string notr="true">... and X more.</string>
```

Reply:
Done